### PR TITLE
Fix small fixes

### PR DESCRIPTION
**Pull request by jianaijun**

### CODE REVIEW

**Review by vangheem — APPROVED** [review]

**Comment by vangheem** [comment]
Thank you!

File: docs/source/developer/fields.md [hunk 1]
```diff
@@ -394,25 +394,6 @@ Clear:
...
-            "op": "assign",
-            "value": {
-                "key": "foo",
-                "value": "bar"
-            }
-        }]
-    }
-}
-```
-
```
Comment (masipcat) [inline docs/source/developer/fields.md:415]: Why this part is deleted?

Comment (jianaijun) [reply docs/source/developer/fields.md:415]: Seems that BucketListField & BucketDictField doesn't support "multi" operations, maybe this is a bug?
 doesn't seem to support xx operations, maybe this is a bug?
`{
    "reason": "deserializationError",
    "details": "Could not deserialize the content",
    "type": "DeserializationError",
    "eid": "630045b5ef734b369e93dd726c8f75d4",
    "message": "DeserializationError()",
    "deserialization_errors": [
        {
            "message": "\"multi\" not a valid operation",
            "field": "field"
        }
    ]
}`

Comment (masipcat) [reply docs/source/developer/fields.md:415]: Sorry, yeah, I thought this was the part of PatchFields

**Pull request merged**